cluster/gce/gci/configure-helper.sh

```diff
@@ -48,6 +48,9 @@ function config-ip-firewall {
     iptables -A FORWARD -w -p UDP -j ACCEPT
     iptables -A FORWARD -w -p ICMP -j ACCEPT
   fi
+  if [[ -n "${BLACKHOLE_MDS}" ]]; then
```

Comment:
Also, is "MDS" a common reduction of "metadata server" ?  It's not in my vernacular...